pyproject.toml

```diff
@@ -50,9 +50,10 @@ format-jinja = """
 files = ["skdecide/__init__.py"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = ">=3.8,<3.12"
```

Comment:
I am wondering is ">=3.8,<3.12" is better than "^3.8" (to avoid changing this line regularly)? I see pros & cons...